src/main/java/duke/services/Storage.java

```diff
@@ -0,0 +1,87 @@
+package duke.services;
+
+import duke.tasks.*;
```

Comment:
Perhaps risky to include all classes in the tasks package despite every member of the package being used.